packages/genesys-spark-components/src/components/stable/gux-calendar/tests/gux-calendar.methods.spec.ts

```diff
@@ -127,10 +127,10 @@ describe('gux-calendar', () => {
         expect(result.length).toEqual(6);
       });
       it('onDateClick with single mode', async () => {
-        spyOn(component, 'setValue').and.callFake(() => {
-          return;
+        jest.spyOn(component, 'setValue').mockImplementation(() => {
+          return Promise.resolve();
         });
-        spyOn(component, 'emitCalendarSelect').and.callFake(() => {
+        jest.spyOn(component, 'emitCalendarSelect').mockImplementation(() => {
```

Comment:
callFake has been removed in favor of mockImplementation when making a custom mock of a function